sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/spanner/changestreams/SpannerChangeStreamErrorTest.java

```diff
@@ -300,9 +330,19 @@ public void testInvalidRecordReceived() {
     mockInvalidChangeStreamRecordReceived(startTimestamp, endTimestamp);
 
     try {
+       final SpannerConfig changeStreamConfig =
+        SpannerConfig.create()
```

Comment:
Positive comment - inlining the config also makes the test better, as the setup should be inline with the result. Great.